atlp-rwanda/e-commerce-lydia-32-bn
# ft: implement user signup and verify their email

### What does this PR do?
Add signup implementation and email verification
### Description of Task to be completed?
Signup a new user and verify their email,
by sending them verification email.
### How should this be manually tested?
1. Clone this repo
2. Open it in you code editor
3. start the server by running `npm run start`
4. navigate to this [link](http://localhost:3000/api/register) on postman
5. Enter all field to verify and token will be provided
6. Navigate to this [link](http://localhost:3000/api/verify) 
and add the token Then the user will be verified

### Any background context you want to provide?
N/A
### Screenshots (if appropriate)
N/A
### Questions:
N/A
### What are the relevant pivotal tracker stories

#187570860
